.github/workflows/build_and_test.yml

```diff
@@ -47,6 +47,9 @@ jobs:
           - image: ubuntu2204
             python: python3
             config-zstd: OFF
+          - image: ubuntu2204-icpx
+            python: python3
+            config-zstd: OFF
```

Comment:
maybe we can add another variable `additional-flags: -DCMAKE_CXX_COMPILER=icpx` and then add it in the `Configure PcapPlusPlus` step?

```yaml
run: cmake -DLIGHT_PCAPNG_ZSTD=${{ matrix.config-zstd }} -DPCAPPP_BUILD_COVERAGE=ON ${{additional-flags}} -S . -B $BUILD_DIR
```